examples/training_scripts/height_example/train_height_config_json.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+import logging
+import time
+from typing import Optional, Dict, Any
+from argparse import ArgumentParser
+
+from syne_tune import Reporter
+from syne_tune.config_space import randint
+from benchmarking.utils import add_config_json_to_argparse, load_config_json
+
+
+report = Reporter()
+
+
+RESOURCE_ATTR = "epoch"
```

Comment:
Cannot import them from `train_height.py`, because `examples` is not a dependence of the SM estimator (I tried).